<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="maxIterationCombobox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="nieghbourSizeCombobox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="increaseTempItersCombobox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>350</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="genderGroupIterCombobox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>450</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="neigbourGeneratorCombobox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="initSolverCombobox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>450</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="startingTemperatureCombobox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="simplexCoolingMulitplierCombobox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="stopTemperatureCombobox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>350</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="prizeSizeCombobox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="maxIterationLabel">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Number of iterations</string>
    </property>
   </widget>
   <widget class="QLabel" name="neighbourSizeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Neigbourhood size</string>
    </property>
   </widget>
   <widget class="QLabel" name="prizeSizeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Prize size</string>
    </property>
   </widget>
   <widget class="QLabel" name="increaseTempIterLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No Cooling iteration</string>
    </property>
   </widget>
   <widget class="QLabel" name="genderGroupLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender Grouper iteration</string>
    </property>
   </widget>
   <widget class="QLabel" name="coolingMultiplier">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string>Cooling Multiplier</string>
    </property>
   </widget>
   <widget class="QLabel" name="startingTemperatureLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Starting Temperature</string>
    </property>
   </widget>
   <widget class="QLabel" name="stopTemperatureLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Temperature</string>
    </property>
   </widget>
   <widget class="QLabel" name="initSolverLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Init Solver</string>
    </property>
   </widget>
   <widget class="QLabel" name="neigboutGeneratorLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Neighbour generator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startGameButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>500</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
